Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -321,8 +321,11 @@ static ASCIILiteral serviceName(const ProcessLauncher::LaunchOptions& launchOpti
     // 1. When the application and system frameworks simply have different localized resources available, we should match the application.
     // 1.1. An important case is WebKitTestRunner, where we should use English localizations for all system frameworks.
     // 2. When AppleLanguages is passed as command line argument for UI process, or set in its preferences, we should respect it in child processes.
+#if PLATFORM(MACCATALYST)
```

Comment:
I don't quite understand why this is guarded. We have plenty of platforms that don't use ExtensionKit yet and still need localizations to propagate, so it is a surprising change!

Reply:
@pvollan halp. can you explain the motivation?

Reply:
I believe the guard was added to reduce risk, since we haven't seen similar regressions on other platforms. But as you say, we still need localizations to propagate when `AppleLanguages` is not passed as a command line argument, so it seems correct to include this for all platforms that does not use EXTENSIONKIT.

Reply:
I see! Yes, and I believe Lily has confirmed impact on other non-EK platforms.